includes/admin/class-wc-admin-setup-wizard.php

```diff
@@ -578,55 +443,257 @@ protected function wc_setup_wcs_tout() {
+			'repo-slug' => 'woocommerce-services',
+		) );
+	}
+
+	/**
+	 * Get the WCS shipping carrier for a given country code.
+	 *
+	 * Can also be used to determine if WCS supports a given country.
+	 *
+	 * @param $country_code
+	 * @return bool|string Carrier name if supported, boolean False otherwise.
+	 */
+	protected function get_wcs_shipping_carrier( $country_code ) {
+		switch ( $country_code ) {
+			case 'US':
+				return 'USPS';
+			case 'CA':
+				return 'Canada Post';
+			default:
+				return false;
+		}
+	}
+
+	/**
+	 * Get shipping methods based on country code.
+	 *
+	 * @param $country_code
+	 * @return array
+	 */
+	protected function get_wizard_shipping_methods( $country_code ) {
+		$shipping_methods = array(
+			'live_rates' => array(
+				'name'        => __( 'Live Rates', 'woocommerce' ),
+				'description' => __( 'Shipping rates updated in realtime. Powered by Jetpack.', 'woocommerce' ),
+			),
+			'flat_rate' => array(
+				'name'        => __( 'Flat Rate', 'woocommerce' ),
+				'description' => __( 'Set a fixed price to cover shipping costs.', 'woocommerce' ),
+				'settings'    => array(
+					'cost' => array(
+						'type'        => 'text',
+						'description' => __( 'What would you like to charge for flat rate shipping?', 'woocommerce' ),
+					),
+				),
+			),
+			'free_shipping' => array(
+				'name'        => __( 'Free Shipping', 'woocommerce' ),
+				'description' => __( "Don't charge for shipping.", 'woocommerce' ),
+			),
+		);
+
+		$live_rate_carrier = $this->get_wcs_shipping_carrier( $country_code );
+
+		if ( false === $live_rate_carrier ) {
+			unset( $shipping_methods['live_rates'] );
+		}
+
+		return $shipping_methods;
+	}
+
+	/**
+	 * Render the available shipping methods for a given country code.
+	 *
+	 * @param string $country_code
+	 * @param string $input_prefix
+	 */
+	protected function shipping_method_selection_form( $country_code, $input_prefix ) {
+		$live_rate_carrier = $this->get_wcs_shipping_carrier( $country_code );
+		$selected          = $live_rate_carrier ? 'live_rates' : 'flat_rate';
+		$shipping_methods  = $this->get_wizard_shipping_methods( $country_code );
+		?>
+		<div class="wc-wizard-shipping-method-select">
+			<select id="<?php echo esc_attr( "{$input_prefix}[method]" ); ?>" name="<?php echo esc_attr( "{$input_prefix}[method]" ); ?>" class="method wc-enhanced-select">
+			<?php foreach ( $shipping_methods as $method_id => $method ) : ?>
+				<option value="<?php echo esc_attr( $method_id ); ?>" <?php selected( $selected, $method_id ); ?>>
+					<?php echo esc_html( $method['name'] ); ?>
+				</option>
+			<?php endforeach; ?>
+			</select>
+
+			<div class="shipping-method-description">
+			<?php foreach ( $shipping_methods as $method_id => $method ) : ?>
+				<p class="<?php echo esc_attr( $method_id ); ?>" <?php if ( $method_id !== $selected ) echo 'style="display:none"'; ?>>
+					<?php echo esc_html( $method['description'] ); ?>
```

Comment:
Should this description be part of the `<option>` text? It feels like it would make things a bit clearer if you see all in the dropdown:
* Free Shipping - Don't charge for shipping
* etc

Reply:
cc: @olaolusoga

Reply:
Let's stick with the description on side. Though "Free shipping" can be described in 4 words, the same can't be said for live rates, flat rates, or local pickup.